src/Game/Game.h

```diff
@@ -110,10 +117,11 @@ class Game
     bool _quit = false;
     SDL_Event _event;
     bool _initialized = false;
+    std::unique_ptr<InputController> inputController_;
```

Comment:
Strictly speaking identifiers with '_' prefix aren't legal in terms of C++.
All of them are reserved for needs of C++ standard library.
That's why I prefer suffix '_'